contentmgr/replication.go

```diff
@@ -550,6 +549,16 @@ func (cm *ContentManager) processStagingZone(ctx context.Context, zone util.Cont
 		break
 	}
 
+	// if not already processing, consolidation wasn't needed
+	if !cm.IsZoneProcessing(zone.ID) {
+		cm.MarkStartedProcessing(zone)
+	}
+	// if already processing, consolidation was needed but is now completed.
+	// we can keep this mark set and jump right into aggregation
+
+	// once aggregation is done, mark as finished processing
+	defer cm.MarkFinishedProcessing(zone)
```

Comment:
nope, aggregation is done when the zone content is pinned - either locally or when the incomplete events comes in from shuttle

Reply:
so remove markers where the pincomplete event is handled and where local aggregation is done

Reply:
lastly, how does this prevent duplicate runs for aggregation?